lib/update-check.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var latestVersion = require('latest-version');
+var semver = require('semver');
+var info = require('../package.json');
+var settings = require('../settings');
+var Spinner = require('cli-spinner').Spinner;
+var chalk = require('chalk');
+Spinner.setDefaultSpinnerString(Spinner.spinners[7]);
+
+function check(next) {
+	console.error('particle-cli v' + info.version);
+	console.error();
```

Comment:
I wouldn't add this new output at the beginning of every CLI invocation. Is this only for debugging on this branch?